templates/parse_file.html

```diff
@@ -72,17 +72,22 @@ <h1 class="title">CAMELOT Parse file {{filename}}</h1>
                     </div>
                 </tr>
                 {% endfor %}
+                </tbody>
+                <thead><tr><th colspan=3><font color="red">*</font>Type of dataset</th></tr></thead>
+                <tbody>
                 <tr><td class="middle">  <input type="radio" name="ObsSim" value="IsObserved"> Observations </td>
-                    <td colspan=2 class="middle">  <input type="radio" name="ObsSim" value="IsSimulated"> Simulations </td></tr>
+                    <td colspan=2 class="middle"> <input type="radio" name="ObsSim" value="IsSimulated"> Simulations </td></tr>
                 <tr><td class="middle">  <input type="radio" name="GalExgal" value="IsGalactic"> Galactic </td>
-                    <td colspan=2 class="middle">  <input type="radio" name="GalExgal" value="IsExtragalactic"> Extragalactic </td></tr>
+                    <td colspan=2 class="middle"> <input type="radio" name="GalExgal" value="IsExtragalactic"> Extragalactic </td></tr>
                 <tr><td colspan=3 class="middle"> <font color="#BDBDBD">(for simulations, choose Galactic unless an entire galaxy is covered)</font> </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="username" name="Username" value="" required> FirstnameLastname of first author </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="email" name="Email" value="" required> Your email address <font color="#BDBDBD">(this will be kept private)</font> </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="adsid" name="adsid" value=""> ADS ID  </td></tr>
-                <tr><td colspan=3 class="middle">  <input type="text" id="doi" name="doi" value="" required> <a href="http://www.doi.org/">DOI: a permanent data link</a> 
-                        or URL <font color="#BDBDBD">(if this is over 64 characters, please <a href="http://tinyurl.com">shorten</a> it
+                <tr><td colspan=3 class="middle"> <input type="text" id="username" name="Username" value="" required> <font color="red">*</font>FirstnameLastname of first author </td></tr>
+                <tr><td colspan=3 class="middle"> <input type="text" id="adsid" name="adsid" value=""> <font color="red">*</font>ADS ID <font color="#BDBDBD">(not the full link)</font> or 
+                                                  <input type="text" id="doi" name="doi" value="" required> <a href="http://www.doi.org/">DOI</a> or URL</td></tr>
```

Comment:
Make this "and" since an entry for both is preferred and an entry for DOI/URL is always required.